src/server/views/lists/partials/translators-interpreters/translators-interpreters-results-list.njk

```diff
@@ -12,16 +12,36 @@
   <p class="govuk-body">We couldn’t find any results that match your search terms. Try changing these to search again.</p>
 {% else %}
   <h2 class="govuk-heading-l govuk-!-margin-bottom-4">{{resultsTitle}}</h2>
-  <p class="govuk-body">You should carry out your own research before choosing a service provider.</p>
 
-  <div class="govuk-warning-text">
-    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-    <strong class="govuk-warning-text__text">
-      <span class="govuk-warning-text__assistive">Warning</span>
-      Translators and interpreters must be authorised to operate in some countries (known as sworn translators).
-      Check providers are qualified to carry out the service you need in the country you have selected.
-    </strong>
-  </div>
+  {% if hasSworn.translatorsAndInterpreters or hasSworn.interpreters or hasSworn.translators %}
```

Comment:
`hasSworn.interpreters or hasSworn.translators` should be fine (no need to check for hasSworn.translatorsAndInterpreters?)